lib/matplotlib/transforms.py

```diff
@@ -1518,28 +1518,26 @@ def transform_path_non_affine(self, path):
 
     def transform_angles(self, angles, pts, radians=False, pushoff=1e-5):
         """
-        Performs transformation on a set of angles anchored at
-        specific locations.
+        Transforms a set of angles anchored at specific locations.
 
-        The *angles* must be a column vector (i.e., numpy array).
-
-        The *pts* must be a two-column numpy array of x,y positions
-        (angle transforms currently only work in 2D).  This array must
-        have the same number of rows as *angles*.
-
-        *radians* indicates whether or not input angles are given in
-         radians (True) or degrees (False; the default).
-
-        *pushoff* is the distance to move away from *pts* for
-         determining transformed angles (see discussion of method
-         below).
-
-        The transformed angles are returned in an array with the same
-        size as *angles*.
-
-        The generic version of this method uses a very generic
-        algorithm that transforms *pts*, as well as locations very
-        close to *pts*, to find the angle in the transformed system.
+        Parameters
+        ----------
+        angles : (N,) array-like
+            The angles to transform.
+        pts : (N, 2) array-like
+            The points where the angles are anchored.
+        radians : bool, default False
+            Whether *angles* are in degrees (the default) or in radians.
```

Comment:
```suggestion
            Whether *angles* are radians or degrees.
```
- It makes more sense to state radians first since this is the name of the parameter.
- We do not have to awkwardly squeeze the default in. It's already stated in the above line.